grpc-sys/Cargo.toml

```diff
@@ -11,8 +11,21 @@ description = "FFI bindings to gRPC c core library"
 categories = ["external-ffi-bindings", "network-programming"]
 build = "build.rs"
 exclude = [
+    "grpc/doc/*",
```

Comment:
Do it require grpc/ ?
 - .github/
 - .vscode/
 - bazel/
 - summerofcode/
 - src/benchmark/
 - src/csharp/
 - etc...